smoke-tests/designer/features/completeAForm.feature

```diff
@@ -0,0 +1,24 @@
+Feature: Components
+  As a forms user
+  I want to complete a form
+  So that I submit my form successfully
+
+  Background: User views the form
+    Given I am at the beginning of the "report a terrorist" form
+
+  # Remove @wip tag when the cookie issue is resolved
+  @wip
```

Comment:
are you still getting this issue when you're running with docker compose (runner + designer + redis)?

Reply:
@jenbutongit it's really odd it works against Heroku yet in docker-compose when running headlessly it fails miserably.